packages/core/theme/src/components/alert.ts

```diff
@@ -0,0 +1,94 @@
+import type {VariantProps} from "tailwind-variants";
+
+import {tv} from "../utils/tv";
+/**
+ * Alert wrapper **Tailwind Variants** component
+ *
+ * @example
+ * ```js
+ * const {base, mainWrapper, title, description} = input({...})
+ *
+ * <div className={base())}>
+ *    // icon
+ *
+ *    <div classsName ={mainWrapper()}>
+ *       <div className={title()}>Title</div>
+ *       <div className={description()}>Description</div>
+ *   </div>
+ *
+ *    <button>
+ *      // close button
+ *    </button>
+ * </div>
+ * ```
+ */
+
+const alert = tv({
+  slots: {
+    base: ["group flex flex-row w-[342px] h-[68px]  p-[12px]"],
+    title: ["text-medium font-normal block h-[24px]"],
+    description: ["text-small font-normal h-[20px]"],
+    mainWrapper: ["w-[268px] h-[44px] ml-[20px] flex flex-col box-border items-start"],
+    closeButton: ["w-[24px h-[24px] cursor-pointer relative"],
```

Comment:
don't use fixed values here.

Reply:
We don't use the [arbitrary values](https://tailwindcss.com/docs/width#arbitrary-values) but TW classes. e.g. `p-[12px]` -> `p-3`, `h-[20px]` -> `h-5`.

Reply:
ok